x-pack/plugins/uptime/public/components/monitor/ping_list/ping_list.test.tsx

```diff
@@ -185,5 +185,23 @@ describe('PingList component', () => {
         expect(rowShouldExpand(ping)).toBe(true);
       });
     });
+
+    describe('formatDuration', () => {
+      it('returns zero for < 1 millisecond', () => {
+        expect(formatDuration(984)).toBe('0 ms');
+      });
+
+      it('returns milliseconds string if < 1 seconds', () => {
+        expect(formatDuration(921_039)).toBe('921 ms');
```

Comment:
TIL, this _ usage

Reply:
Isn't it great? There's a caveat, it only seems to work in test code. Kibana runtime errors on it, so there's probably some feature that needs enabled if we were going to use it in production code.